src/tqec/compile/tree/tree.py

```diff
@@ -238,13 +250,23 @@ def _generate_annotations(
         k: int,
         manhattan_radius: int = 2,
         detector_database: DetectorDatabase | None = None,
+        database_path: Path = Path(DEFAULT_DETECTOR_DATABASE_PATH),
```

Comment:
The `DEFAULT_DETECTOR_DATABASE_PATH` should already be a `Path` instance. It is better to declare it as a `Path` and not instantiate a new instance each time this method is called.